.travis.yml

```diff
@@ -2,11 +2,6 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-notifications:
```

Comment:
Accidental change? Shouldn't someone get email for failures?

Reply:
Bad merge, maybe. Reverted.